src/trunk/apps/messaging/scmaster/config/scmaster.py

```diff
@@ -346,7 +346,7 @@ def updateConfig(self):
 
         print("OK", file=sys.stderr)
 
-        version = out[0].strip()
+        version = out[0].decode(sys.stdout.encoding).strip()
```

Comment:
Wouldn't `sys.stderr.encoding` be the right stream here as we are going to write to stderr? This is really just out of curiosity, I am not very familiar with the Python3 string encoding/decoding stuff.